appserver/node-express/lib/db-client/contributor/index.js

```diff
@@ -6,11 +6,14 @@ var util = libRequire('db-client/util');
 
 var funcs = {};
 
-/*
- * REPUTATION CHANGE
- * A contributor's reputation may be changed up or down --
- * repChange may be a positive or negative int. Use server-side
- * math so we don't need to read the value into the middle-tier
+/**
+ * Handle an update to a contributor's reputation. This occurs during
+ * a question vote, an answer vote, or an answer acceptance.
+ *
+ * @param  {String} txid The transaction ID.
+ * @param  {String} id The contributor ID.
+ * @param  {Number} repChange Increment (1) or decrement (-1) value.
```

Comment:
+10 would work as would -3. Let's document this function as abstractly as its written, where repChange is expected to be a signed integer. Other functions are what worry about setting the value. This one intentionally just does addition of a (positive or negative) number.